canonical-web-and-design/juju.is
# About operator lifecycle manager

## QA
- Go to http://0.0.0.0:8041/about/operator-lifecycle-manager
- Check the content matches [the copy doc](https://docs.google.com/document/d/1B0adX3iTzNrF_RORQUWU6MaYn_E2wzvfJYUbolAY3Ok/edit)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/104
